ecosystem/resources/members/povm-toolbox.toml

```
@@ -5,7 +5,8 @@ licence = "Apache License 2.0"
 contact_info = "oss@zurich.ibm.com"
 alternatives = "_No response_"
 affiliations = "_No response_"
-labels = [ "Quantum information",]
+labels = []
```

Why this change?

---

Good spot, that does seem to be a mistake.

```suggestion
labels = [ "Quantum information",]
```

---

Turns out I accidentally removed "Quantum information" in the automated step. Fortunately it only affects two projects.